### ISSUE TITLE
[Feature request] Block internet connection for a device

### ISSUE THREAD

**Issue by CannotTouch** [Trigus42/FritzBox-UPNP#2]
Hi,
I wish suggest to insert the possibility to block/unblock the internet connection for a specific device.

The option is under #kidLis Internet--Filters---block device

Thanks for support

**Comment by Trigus42** [comment]
I think I found a library that seems to do exactly what you want to archive:
https://github.com/blacksenator/fritzsoap/blob/8393cbb7bd20ca14201c6c802d4fee69d4b34b1f/src/x_hostfilter.php#L85-L102

I don't feel comfortable adding this and possibly more functionality to this small project with the way it is currently structured. It would take quite some work to turn this project into something easily expandable and maintainable.  
Also, when I created this project, I didn't know of fritzsoap, but now it would feel redundant to put in much work here.

If you would like to maintain this project, however, let me know. Or, if you want to fork it, feel free to so.
